### ISSUE TITLE
Can't connect new API to Bitbucket repository

### ISSUE THREAD

**Issue by bnelissen-orisha** [a85/POSTMan-Chrome-Extension#11341]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

I can't create a new API with Bitbucket repository, It says that another API is using it even if there no other API.

### Steps To Reproduce

1. Go to API tab
2. Click on "Create an API"
3. Choose Bitbucket and confirm authorization
4. Choose Bitbucket workspace
5. Choose repository
6. Can't click on repository because of this : "You can't select a repository which is already used by another API"

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/37617999/194826839-3fb0d36a-c566-455b-817e-29c111428297.png)

### Operating System

Windows 11 22H2

### Postman Version

10.0.20

### Postman Platform

Both

**Comment by abhijitkane** [comment]
@bnserig While we look into this, can you reach out to us at help@postman.com with the name of the repository you want to link to? We should be able to unblock setting up the integration.

**Comment by shashankawasthi88** [comment]
@bnserig Can you confirm that the repo you are trying to connect is not connected to any other API  in your team's account?

**Comment by bnelissen-orisha** [comment]
@shashankawasthi88 I don't know if a colleague of my team has an API connected to this repo.
I tried to leave the team (and it erase all my data...) but I can't signon anymore in my account. I says : "Your team only supports Postman v9". It's broken, I don't know what to do.

**Comment by shashankawasthi88** [comment]
@bnserig can you reach out to [help@postman.com](mailto:help@postman.com), we will try to unblock you on this flow. @postman-support ^

**Comment by LubomirS** [comment]
We have the same issue. An Api is connected to Github repository in one personal workspace, and a second person on their own personal workspace cannot connect the Api to Github repository, even though they have not connected any other api to any Github repository.

**Comment by shashankawasthi88** [comment]
@bnserig @LubomirS can you reach out to our support team? We can help try to figure out which API is connected to that particular repo.
@LubomirS also which plan are you on? For the free team, there is only one connection allowed.

**Comment by LubomirS** [comment]
@shashankawasthi88 We are on a professional plan. However, we are going to sidestep the issue by creating a private team and invite all the members, instead of having all members connect api to repository in their private workspace, which is probably a better way to handle our use case. So this is not a blocking issue in any way. Thanks for reaching out anyway.

**Comment by shashankawasthi88** [comment]
@LubomirS yeah, the APIs are meant to be collaborative in nature. One member in the team can go ahead, setup an API and the git connection. The roles can be assigned on that API and the API editors can go about making changes to the API, pushing, pulling from the repository etc. You can schedule a time with me using [this](https://calendly.com/shashank-awasthi/v10-feedback) link. We can discuss in detail your workflow and how it can be catered best in the product.

**Comment by shashankawasthi88** [comment]
Hi,

As of [Postman v12](https://www.postman.com/downloads/), we now support Native Git Integration.
You can connect a workspace directly to a local Git repository and work against your filesystem. Collections and related artifacts are stored as files, enabling standard Git workflows (branching, diffs, PRs, CI/CD) without manual export/import.
In the new native git integration, one workspace connects to one repositories. If a member in the team tries to connect another workspace to the same repo, they are redirected to the existing workspace that was already connected, easing the discovery of the collaborative workspace. Feel free to try out the integration and provide feedback.

Docs: https://learning.postman.com/docs/agent-mode/native-git
Announcement: https://blog.postman.com/new-postman-is-here/

https://github.com/user-attachments/assets/d8fa28ef-7a7d-4bc4-98aa-159f96582f4f
